code/modules/modular_computers/file_system/programs/minesweeper.dm

```diff
@@ -286,38 +291,45 @@
 	var/result = select_square(x,y)
 	game_status = result
 	if(result == MINESWEEPER_VICTORY)
-		play_snd('yogstation/sound/arcade/minesweeper_win.ogg')
-		host.say("You cleared the board of all mines! Congratulations!")
-		if(emaggable && host.obj_flags & EMAGGED && value >= 1)
+		winner_chicken_dinner(user)
+
+	if(result == MINESWEEPER_DEAD && emaggable && (host.obj_flags & EMAGGED))
+		// One crossed wire, one wayward pinch of potassium chlorate, ONE ERRANT TWITCH
+		// AND
+		KABLOOEY()
+	
+	if(result)
+		time_frozen = REALTIMEOFDAY - starting_time
+
+	return TRUE
+
+/datum/minesweeper/proc/winner_chicken_dinner(mob/user)
+	play_snd('yogstation/sound/arcade/minesweeper_win.ogg')
+	host.say("You cleared the board of all mines! Congratulations!")
+	if(emaggable && host.obj_flags & EMAGGED && value >= 1 && !emagwin)
```

Comment:
non-emag winners do not appear to get any ticket_count anymore. accidental?

Reply:
Fixed with latest push